docassemble/EFSPIntegration/efm_client.py

```diff
@@ -361,17 +362,14 @@ def get_service_contact(self, service_contact_id):
     send = lambda: self.proxy_client.get(self.base_url + f'firmattorneyservice/service-contacts/{service_contact_id}')
     return self._call_proxy(send)
 
-  def update_service_contact(self, service_contact_id, **kwargs):
+  def update_service_contact(self, service_contact_id, service_contact:Individual):
+    service_contact_dict = serialize_person(service_contact)
     send = lambda: self.proxy_client.patch(self.base_url + f'firmattorneyservice/service-contacts/{service_contact_id}', 
-        data=json.dumps(kwargs))
+        data=json.dumps(service_contact_dict))
     return self._call_proxy(send)
 
-  def create_service_contact(self, service_contact:Individual, is_public:bool, is_in_master_list:bool, admin_copy:str=None):
+  def create_service_contact(self, service_contact:Individual):
```

Comment:
We still need these extra parameters when creating a service. Was this just a merge issue, or did you intentionally remove them?

Reply:
not something I remember touching so probably a merge issue--no objection to restoring those. interestingly it worked without but i will add those parameters as questions in the interview